### PR TITLE
COR-512: offer template download

### PR DESCRIPTION
**Pull request by internetti**
added a download link for the template in the navigation

### CODE REVIEW

File: web/templates/nav.gohtml [hunk 1]
```diff
@@ -11,7 +11,10 @@
             <div id="navbarContent">
                 <div class="me-auto mb-2 mb-lg-0 w-100">
                     <div class="d-flex flex-lg-row flex-column w-100">
-                        <a class="nav-link" href="/countries">
+                        <a href="/static/template.xlsx" download="nrc-grf-template" class="nav-link">
+                            Download template
+                        </a>
+                        <a class="nav-link ms-lg-5 ms-0" href="/countries">
```
Comment (neb42) [inline web/templates/nav.gohtml:17]: There's another pr that changes the margins from `ms-lg-5 ms-0` to `ms-5`. May as well do it here now.

File: web/templates/nav.gohtml [hunk 2]
```diff
@@ -11,7 +11,10 @@
             <div id="navbarContent">
                 <div class="me-auto mb-2 mb-lg-0 w-100">
                     <div class="d-flex flex-lg-row flex-column w-100">
-                        <a class="nav-link" href="/countries">
+                        <a href="/static/template.xlsx" download="nrc-grf-template" class="nav-link">
```
Comment (neb42) [inline web/templates/nav.gohtml:14]: Maybe call the file `nrc_grf_template.xlsx`

**Review by neb42 — COMMENTED** [review]
Maybe create a jira ticket to populate the fields in the template. Or add it as a requirement to one of the existing tickets around setting fields.

**Comment by ludydoo** [comment]
lgtm

**Comment by neb42** [comment]
lgtm

**Comment by neb42** [comment]
I'll merge as Irena is off today

**Pull request merged**